android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -299,7 +299,9 @@ public void removeJSComponentEvents(ReadableArray events) {
     public void sendSignal(ReadableMap signal, Callback callback) {
 
         Session mSession = sharedState.getSession();
-        mSession.sendSignal(signal.getString("type"), signal.getString("data"));
+        if (mSession != null){
+            mSession.sendSignal(signal.getString("type"), signal.getString("data"));
+        }
         callback.invoke();
```

Comment:
We should `invoke` an error if we are unable to send a signal and modify the JS code to adapt as well.